pkg/output/table.go

```diff
@@ -68,6 +93,8 @@ func ToTable(resources []*schema.Resource) ([]byte, error) {
 
 	t.Render()
 
+	showSkippedResourcesTable(resources, c.Bool("show-skipped"), bufw)
```

Comment:
Could we do something like this:
```go
bufw.WriteString(skippedResourcesMessage(resources, c.Bool("show-skipped")))
```
and put the logic for the message into a shared function in `output/output.go`?

Reply:
Done